<ui version="4.0" >
 <class>FormPersonel</class>
 <widget class="QWidget" name="FormPersonel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Personel</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&amp;Ara :</string>
     </property>
     <property name="buddy" >
      <cstring>arama_kutusu</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="arama_kutusu" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTableView" name="personel_tablo" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>172</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>253</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>210</red>
           <green>218</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>111</red>
           <green>121</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>211</red>
           <green>218</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>253</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>210</red>
           <green>218</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>111</red>
           <green>121</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>211</red>
           <green>218</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>253</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>210</red>
           <green>218</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>111</red>
           <green>121</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>83</red>
           <green>90</green>
           <blue>93</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>167</red>
           <green>181</green>
           <blue>186</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="mouseTracking" >
      <bool>true</bool>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="tabKeyNavigation" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle" >
      <enum>Qt::DashLine</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Künye Bilgileri</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Sicil No:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="LeSicilNo" />
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Role No:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QToolButton" name="TbRoleSec" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" >
        <widget class="QLineEdit" name="LeRoleNo" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Ad:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="LeAd" />
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Soyad:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="2" >
        <widget class="QLineEdit" name="LeSoyad" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Sınıf:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="CbSinif" />
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Rütbe:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2" >
        <widget class="QComboBox" name="CbRutbe" />
       </item>
       <item row="3" column="0" colspan="5" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="title" >
          <string>İşlemler</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QPushButton" name="pBDegistir" >
            <property name="text" >
             <string>Değiştir</string>
            </property>
            <property name="icon" >
             <iconset resource="ikon.qrc" >:/ikonlar/cnr-update.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="pBKaydet" >
            <property name="text" >
             <string>Yeni Kayıt</string>
            </property>
            <property name="icon" >
             <iconset resource="ikon.qrc" >:/ikonlar/filesaveas.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="pBiptal" >
            <property name="text" >
             <string>iPtal</string>
            </property>
            <property name="icon" >
             <iconset resource="ikon.qrc" >:/ikonlar/edit_remove.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QPushButton" name="pBSil" >
            <property name="text" >
             <string>Sil</string>
            </property>
            <property name="icon" >
             <iconset resource="ikon.qrc" >:/ikonlar/eraser.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Görevler</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTableView" name="tablo_mevcut_gorevler" >
         <property name="contextMenuPolicy" >
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Adres Bilgileri</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QFrame" name="adreslerFrame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSil" >
   <property name="icon" >
    <iconset resource="ikon.qrc" >:/ikonlar/eraser.png</iconset>
   </property>
   <property name="text" >
    <string>Sil</string>
   </property>
  </action>
  <action name="actionGorevSil" >
   <property name="icon" >
    <iconset resource="ikon.qrc" >:/ikonlar/eraser.png</iconset>
   </property>
   <property name="text" >
    <string>GorevSil</string>
   </property>
  </action>
  <action name="actionGorev_Ekle" >
   <property name="icon" >
    <iconset resource="ikon.qrc" >:/ikonlar/onayla.png</iconset>
   </property>
   <property name="text" >
    <string>Gorev Ekle</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>arama_kutusu</tabstop>
  <tabstop>personel_tablo</tabstop>
 </tabstops>
 <resources>
  <include location="ikon.qrc" />
 </resources>
 <connections/>
</ui>
